### PR TITLE
Add tests for the bootloader

### PR DESCRIPTION
**Pull request by poncovka**
Added new tests for the bootloader options.

### CODE REVIEW

File: bootloader-2.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+. ${KSTESTDIR}/functions.sh
+
+prepare_disks() {
+    tmpdir=$1
+
+    qemu-img create -q -f qcow2 ${tmpdir}/disk-a.img 10G
+    qemu-img create -q -f qcow2 ${tmpdir}/disk-b.img 10G
+    echo ${tmpdir}/disk-a.img ${tmpdir}/disk-b.img
+}
```
Comment (jkonecny12) [inline bootloader-2.sh]: Please add a new line here.

File: bootloader-2.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (C) 2017  Red Hat, Inc.
```
Comment (jkonecny12) [inline bootloader-2.sh]: Please fix the date here.

File: bootloader-3.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+. ${KSTESTDIR}/functions.sh
+
+prepare_disks() {
+    tmpdir=$1
+
+    qemu-img create -q -f qcow2 ${tmpdir}/disk-a.img 10G
+    qemu-img create -q -f qcow2 ${tmpdir}/disk-b.img 10G
+    echo ${tmpdir}/disk-a.img ${tmpdir}/disk-b.img
+}
```
Comment (jkonecny12) [inline bootloader-3.sh]: Please add a newline here.

**Comment by poncovka** [comment]
Updated.

**Pull request merged**